### PR TITLE
support different languages in url paths, fixes #198

### LINKED ISSUE #198: Support different languages in url paths

### PR DESCRIPTION
**Pull request by fossecode**

### CODE REVIEW

**Review by michaelmcmillan — APPROVED** [review]
Great!

**Comment by Kilian** [comment]
the changes made to the templates will get merge conflicts with the translation branch but I'll fix that.

File: app/domain/urls.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  nl: {
+    submitReport: '/',
+    profile: '/healthcondition',
+    privacyPolicy: '/privacy-policy',
+    map: '/map',
+    contributors: '/contributors',
+    api: '/api',
+    apiDocs: '/api-docs',
+    statistics: '/statistics'
```
Comment (Kilian) [inline app/domain/urls.ts]: translations:

submitReport: '/',
    profile: '/gezondheidstoestand',
    privacyPolicy: '/privacy-policy',
    map: '/kaart',
    contributors: '/bijdragers',
    api: '/api',
    apiDocs: '/api-docs',
    statistics: '/statistieken'

File: app/views/pages/report.ejs [hunk 2]
```diff
@@ -132,7 +128,7 @@
         <h1><%= profile ? 'Din helsetilstand' : 'Rapporter din helsetilstand' %></h1>
         <% if(locals.success) {%>
           <div class="alert alert-success" role="alert">
-            Takk for ditt svar! Benytt denne lenken hvis helsetilstanden din endrer seg: <a target="_blank" href="/helsetilstand/<%= passcode %>">coronastatus.no/helsetilstand/<%= passcode %></a>. 
+            Takk for ditt svar! Benytt denne lenken hvis helsetilstanden din endrer seg: <a target="_blank" href="<%= urls.profile %>/<%= passcode %>">coronastatus.no/helsetilstand/<%= passcode %></a>.
```
Comment (Kilian) [inline app/views/pages/report.ejs]: missed one in the <a> content here :) or should that be fully translated separately?

edit: Lets keep all local urls outside of translation files.

**Review by Kilian — CHANGES_REQUESTED** [review]

File: app/domain/urls.ts (hunk 1 above)
Comment (fossecode) [reply app/domain/urls.ts]: https://github.com/BustByte/coronastatus/pull/199/commits/0077913f275611884107f7380b43039ba6e51ca4

File: app/views/pages/report.ejs (hunk 2 above)
Comment (fossecode) [reply app/views/pages/report.ejs]: Seems like I forgot a few 😅 Think I got them all now, https://github.com/BustByte/coronastatus/pull/199/commits/0077913f275611884107f7380b43039ba6e51ca4

**Review by Kilian — APPROVED** [review]

**Pull request merged**